xcodeproj/internal/xcodeproj.bzl

```diff
@@ -280,7 +280,21 @@ _xcodeproj = make_xcodeproj_rule()
 def xcodeproj(*, xcodeproj_rule = _xcodeproj, **kwargs):
     testonly = kwargs.pop("testonly", True)
 
+    project = kwargs.get("project_name", kwargs.get("name"))
+
+    if kwargs.get("toplevel_cache_buster"):
+        fail("`toplevel_cache_buster` is for internal use only")
```

Comment:
Should this attribute be prefixed with an underscore to highlight it’s internal/private scope?

Reply:
Underscores for attributes mean they are implicit, and I can't pass in a value. I think this error is good enough. I can add a doc comment to the attribute itself so when we generate docs it says it there too.